CoreScriptsRoot/CoreScripts/NotificationScript2.lua

```diff
@@ -475,9 +607,9 @@ function onGameSettingsChanged(property, amount)
 		-- value of 0 is automatic, we do not want to send a notification in that case
 		if level > 0 and level ~= CurrentGraphicsQualityLevel and GameSettings.SavedQualityLevel ~= Enum.SavedQualitySetting.Automatic then
 			if level > CurrentGraphicsQualityLevel then
-				sendNotifcation("Graphics Quality", "Increased to ("..tostring(level)..")", "", 2, nil)
+				sendNotification("Graphics Quality", "Increased to ("..tostring(level)..")", "", 2, nil)
 			else
-				sendNotifcation("Graphics Quality", "Decreased to ("..tostring(level)..")", "", 2, nil)
+				sendNotification("Graphics Quality", "Decreased to ("..tostring(level)..")", "", 2, nil)
```

Comment:
Ah, this wasn't my code. My only change here is correcting a typo in the spelling of "notification" throughout the file. Should I go ahead and fix that up anyway?